test/WebJobs.Script.Tests/ScriptHostTests.cs

```diff
@@ -1600,7 +1600,6 @@ public async Task Initialize_LogsWarningForExplicitlySetHostId()
 
         [Theory]
         [InlineData("python", "main.py", "python", "python")]
-        [InlineData("dotnet-isolated", "app.dll", "dotnet-isolated", "dotnet-isolated")]
```

Comment:
The dotnet isolated worker config was updated to load only in placeholder mode. Few other tests, I was able to set the environment variable and that allows loading the dotnet isolated worker config. But for this test, there are other code path for which this test is running, where we return if we are in placeholder mode. 

Considering the fact that, we are in the process of making this app setting a mandatory value, I think it may be okay to remove the "dotnet-isolated" case from this test.